docs/cli-existing-project.md

```diff
@@ -120,11 +120,11 @@ contract MyDeploymentScript is
   // ...
 ```
 
-### c. Update your `run()` function
+### c. Add the `sphinx` modifier
 
-The entry point of your deployment script must be a `run()` function; it cannot be named anything else. Please change its name if necessary.
+Navigate to the entry point function in your deployment script. This is typically a `run()` function.
 
-Then, add a `sphinx` modifier to your `run` function:
+Then, add a `sphinx` modifier to this function. For example:
```

Comment:
Since we're mentioning the possibility that the entry point is not the run function. We should update the `Propose on testnets` section to cover what you would need to do if you're using an entry point other than run. 

If we don't, people will assume they can use other entry points and immediately run into an error when attempting to propose. This is made worse because we won't immediately have good handling for the error they are likely to run into.

Reply:
Yeah good call. Added this